R/html_notebook.R

```diff
@@ -0,0 +1,204 @@
+#' Convert to an HTML notebook
+#'
+#' Format for converting from R Markdown to an HTML notebook.
+#'
+#' @inheritParams html_document
+#' @param knitr A list of \code{knitr} hooks, used to annotate notebook output.
+#'   See \code{knitr_options} for more information.
+#' @param ... Optional arguments to be passed to \code{\link{html_document}}.
+#' @export
+html_notebook <- function(code_folding = "show",
+                          highlight = "textmate",
+                          knitr = knitr_options_notebook(),
```

Comment:
Does these need exposure here? (there are the yaml-level options, not sure if users would ever tweak these).